docs/_includes/extra/script.js

```diff
@@ -2,6 +2,26 @@
 *  doesn't work due to fixed body height when menu is toggled.
 *  This script intercepts clicks on links, toggles the menu off
 *  and performs the anchor navigation. */
+
+window.addEventListener("load", function () {
+    let href = window.location.pathname;
+    const hash = window.location.hash;
+    if (hash!=="") {
+        href=hash
+    }
+    let element = document.querySelector(`a[href="${href}"]`);
```

Comment:
should we look only for navigation bar instead of links?

Reply:
`querySelector()` returns element or null, never return undefined

Reply:
As for me, it is better to check links because it has a unique identifier as href, and the logic is simpler because we can get a path or anchor from the URL and find this element for scrolling to it. The height of the link element is the same as the li element (navigation bar), so the scrolling would work as expected.

Reply:
Could it be that page content will contain a link with equal href value and script will scroll to it instead? Could `querySelector` return more than one obj? Would it take less time to scan objects only inside navigation bar instead scanning the whole document?

Reply:
sorry, missed this. You are right it is better to search inside the sidebar container. `querySelector()` always returns the first element by provided selector.